delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/indexing/BlazegraphIndexingCoordinator.scala

```diff
@@ -0,0 +1,167 @@
+        stream = eLog
+                   .filterMessage(res =>
+                     (view.resourceTypes.isEmpty || res.types
+                       .union(view.resourceTypes)
+                       .nonEmpty)
+                       && (view.resourceSchemas.isEmpty || view.resourceSchemas.contains(
+                         res.schema.iri
+                       )) && (view.includeDeprecated || res.deprecated)
+                   )
+                   .flatMapMessage { msg =>
+                     msg.value.value.toGraph
+                       .map { graph =>
+                         if (view.includeMetadata) {
+                           val types: Set[Triple] = msg.value.types.map((msg.value.id, rdf.tpe, _))
+                           graph
+                             .add(types)
+                             .add(nxv.rev.iri, msg.value.rev)
+                             .add(nxv.deprecated.iri, msg.value.deprecated)
+                             .add(nxv.createdAt.iri, msg.value.createdAt)
+                             .add(nxv.updatedAt.iri, msg.value.updatedAt)
+                             .add(nxv.updatedBy.iri, msg.value.updatedBy.id)
+                             .add(nxv.createdBy.iri, msg.value.createdBy.id)
+                             .add(nxv.schemaId.iri, msg.value.schema.iri)
+                         } else
+                           graph
+                       }
+                       .flatMap(_.toNTriples)
+                       .flatMap { nTriples =>
+                         (msg.value.id / "graph").toUri
+                           .leftMap(_ => InvalidIri)
+                           .map(
+                             SparqlWriteQuery
+                               .replace(_, nTriples)
+                           )
+                       }
+                       .fold(msg.failed(_), msg.as)
+                   }
+                   .groupWithin(config.client.indexingBulkSize, config.client.indexingBulkMaxWait)
+                   .discardDuplicates()
+                   .runAsyncUnit { bulk =>
+                     client
+                       .bulk(namespace, bulk)
+                       .retryingOnSomeErrors(retryWhen)
+                       .hideErrorsWith(err => new IllegalArgumentException(err.toString()))
+                   }
+                   .discardDuplicatesAndFlatten()
+                   .map(_.void)
+                   .persistProgress(initialProgress, projection, config.persist)
+      } yield stream
+    }
+
+    val indexingRetryStrategy =
+      RetryStrategy[Throwable](config.indexing.retry, _ => true, logError(logger, "blazegraph indexing"))
+
+    for {
+      coordinator <-
+        Task.delay(
+          IndexingStreamCoordinator(projection, idF, buildStream, indexingRetryStrategy, "BlazegraphViewsCoordinator")
+        )
+      _           <- startIndexing(views, coordinator, config)
+    } yield coordinator
+  }
+
+  private def startIndexing(
```

Comment:
We could use the cache for blazegraph views to start the indexing maybe ?
Or we already have one process which is scanning views for it and we could reuse it ?
Or combine both in some way ?

Reply:
We have a process which is getting events by `blazegraph` tag, fetching its latest resource and then indexing it in ddata. We can use the same process to also start/stop a coordinator